docs/sources/flow/setup/install/docker.md

```diff
@@ -21,8 +21,7 @@ Grafana Agent Flow is available as Docker images on the following platforms:
 
 * Ensure that [Docker][] is installed and running on your machine.
 
-* Ensure that you have an existing Grafana Agent Flow configuration file to
-  use saved on your host system, such as:
+* Ensure that you have an existing Grafana Agent river configuration file on your host system, such as:
```

Comment:
River should be a proper noun a la YAML or JSON:

```suggestion
* Ensure that you have an existing Grafana Agent River configuration file on your host system, such as:
```

Reply:
Fixed the typo